### DISCUSSION TITLE
Change the name of The Crate to match Classic.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#4817] [Suggestions]
Currently [Foror's Crate of Endless Resist Gear Storage](https://wowgaming.altervista.org/aowow/?item=23162) is named Crate of Endless Resist Gear Storage on the live server, and remains its original name on the test realm (last I checked). I contend that it should instead be renamed to what Blizzard named it; [A Very Large Bag](https://www.wowhead.com/wotlk/item=23162), as it is, by definition, blizzlike. I also say the rename should apply to the test realm as well as live, but that's pretty secondary.

**Comment by 55Honey** [comment]
Hi @heyitsbench 
I don't have a strong opinion about the names of the bags personally. Feel free to provide a PR here: https://github.com/55Honey/UnsavouryAlexBegone
I'll bring it up with staff then :)

**Reply by heyitsbench** [reply]
> Feel free to provide a PR here

Done. Thank you for pointing it out. ❤️

**Comment by BeefSteaker** [comment]
Should definitely be consistent on the PTR. 
Just to remove all trace of unsavoury names. 

They could instead be replaced by some big names in Azerothcore. To honour people that have put a lot of work in

**Comment by cachocarne** [comment]
https://wowwiki-archive.fandom.com/wiki/Foror's_Crate_of_Endless_Resist_Gear_Storage

AFAIK that was the name give to it when it first appeared, 1.9 PTR for premades testing Naxx, and stayed that way also on 2.2 PTR

**Reply by heyitsbench** [reply]
Yes, the crate was originally named Foror's Crate etc, but it was changed both on CC and live Wow when allegations against the person its name was related to were raised. This post is asking for the rename to be the same between CC and live.
